lib/index.js

```diff
@@ -5,7 +5,9 @@ const isNullOrUndefined = require('./helpers/is-null-or-undefined');
 const VNodeFlags = require('./flags');
 
 function handleWhiteSpace(str) {
-	return str.replace(/[\s\t]{2,}|[\r\n]|[\s\t]+?=[\s\t]/g, '');
+	const twoOrMoreWhiteSpaces = /\s{2,}/g;
+	return str.replace(twoOrMoreWhiteSpaces, '');
```

Comment:
Please remove the constant declaration. The expression isn't used more than once. :)